common/helper.go

```diff
@@ -171,3 +171,16 @@ func SprintfList(list []string, values ...interface{}) (result []string, err err
 	}
 	return
 }
+
+func PathMatch(path string, patternList []string) (bool, error) {
+	for _, pattern := range patternList {
+		result, err := filepath.Match(pattern, path)
```

Comment:
I was wondering if we should resolve symlinks before we do the path matching. What do you think?

Reply:
We could try to resolve links but that makes debugging for the user a bit harder, I will try.

Thinking about it there is maybe another thing we have to validate: The rendered configurations should never be in the `collector_binaries_path` otherwise it would be possible to write a file and execute it (just the execution bit is in between), fully from a remote position.